project/Build.scala

```diff
@@ -258,14 +258,15 @@ object ScaldingBuild extends Build {
 
   lazy val scaldingParquet = module("parquet").settings(
     libraryDependencies ++= Seq(
-      "com.twitter" % "parquet-cascading" % "1.4.0",
+      "com.twitter" % "parquet-cascading" % parquetVersion,
+      "com.twitter" % "parquet-avro" % parquetVersion,
```

Comment:
I think it would make more sense to split into a separate sub module per format, so that dependencies don't grow here.